sbc_compassion/models/import_letter_line.py

```diff
@@ -25,8 +25,6 @@ class ImportLetterLine(models.Model):
 
     sponsorship_id = fields.Many2one('recurring.contract', 'Sponsorship',
                                      compute='_set_sponsorship_id')
```

Comment:
Old code written when I did not know well enough odoo.
Now it is more easy (I am just using the bool return by the fields for knowing if they exist)